var/spack/repos/builtin/packages/busybox/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Busybox(Package):
```

Comment:
```suggestion
class Busybox(MakefilePackage):
```

Reply:
Thanks, I have already refine it in the new commit.